packages/core/src/__tests__/ceramic-api.test.ts

```diff
@@ -431,14 +433,26 @@ describe('Ceramic API', () => {
                    notDoc: '123' },
         metadata: { controllers: [controller] }
       })
+
+      // test data for the atTime feature
+      docFStates.push(docF.state)
+      docFTimestamps.push(Date.now())
```

Comment:
is this adding a timestamp to an update that was never anchored?

Reply:
No this is just a timestamp for the points of times before and in between the actual anchor commits.

Reply:
Now I'm more confused.  I thought only AnchorCommits should have timestamps

Reply:
They do, but here I'm just capturing the time before and after each anchor commit timestamp.